src/Form/AdminSettingsForm.php

```diff
@@ -106,6 +107,19 @@ private function buildCredentialForm() {
       '#default_value' => $credential_settings['site_id'],
       '#required' => TRUE,
     ];
+    $form['public_key'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Public Key'),
+      '#description' => 'Your Acquia Lift Public Key.',
+      '#default_value' => $credential_settings['public_key'],
+      '#required' => TRUE,
+    );
+    $form['secret_key'] = array(
+      '#type' => 'password',
+      '#description' => 'Your Acquia Lift Secret Key. Only required when setting it up for the first time or when you are changing the secret key.',
+      '#title' => t('Secret Key'),
+      '#default_value' => $credential_settings['secret_key'],
+    );
```

Comment:
Before those two were deleted last time, there were more logic to protect especially the secret key. They still exist in [8.x-1.x](https://github.com/acquia/acquia_lift/blob/8.x-1.x/src/Form/AdminSettingsForm.php)